### PR TITLE
New in Chrome 87 post migration

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- First attempt at migrating New in Chrome posts from d.g.c to d.c.c
-
-

### CODE REVIEW

File: site/en/blog/new-in-chrome-87/index.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/xFjaDDkGd2mBReqdPxdy.png'
+alt: >
+  New in Chrome hero logo
+---
+
+Chrome 87 is starting to roll out to stable now.
+
+{% youtube id='NCKMMzVn1c8' %}
```
Comment (petele) [inline site/en/blog/new-in-chrome-87/index.md:20]: Is there any way to get this to appear half-width on wide screen (similar to https://developers.google.com/web/updates/2020/11/nic87)

File: site/en/blog/new-in-chrome-87/index.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+* [Range requests](#rr) and service workers don’t require as many workarounds.
+* The [font access API](#font-access) starts it’s origin trial.
+* And, there’s [plenty more](#more).
+
+I’m [Pete LePage](https://twitter.com/petele), working, and shooting
+from home, let’s dive in and see what’s new for developers in Chrome 87!
+
+## Chrome Dev Summit {: #cds }
+
+{% img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/M6GzgK34vmmwUhsymAmI.png", alt="Chrome Dev Summit logo" %}
```
Comment (petele) [inline site/en/blog/new-in-chrome-87/index.md:35]: How do I get this to appear half-width, left side? I tried `className="w-figure--inline-left"` but that didn't work.

File: site/en/blog/new-in-chrome-87/index.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+});
+```
+
+For an explanation of the issues with range requests and what's changed in
+Chrome 87, see Jeff’s article [Handling range requests in a service worker][wd-rr]
+on web.dev.
+
+## Origin Trial: Font access API {: #font-access }
+
+{% img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/x5qv7Ylxb9i2mtWkBwVt.png", alt="Screen shot of Photopea image editor" %}
```
Comment (petele) [inline site/en/blog/new-in-chrome-87/index.md:154]: How do I get this to appear half-width, right side? I tried `className="w-figure--inline-right"` but that didn't work.

File: site/en/blog/new-in-chrome-87/index.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+  plenty more. Let’s dive in and see what’s new for developers in Chrome 87!
+layout: 'layouts/blog-post.njk'
+date: 2020-11-17
+updated: 2020-11-23
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/xFjaDDkGd2mBReqdPxdy.png'
+alt: >
+  New in Chrome hero logo
+---
```
Comment (robdodson) [inline site/en/blog/new-in-chrome-87/index.md:16]: @petele could you add:

```
tags:
  - chrome-87
```

Chrome tags are special and you don't need to add them to supportedTags.json.

Comment (petele) [reply site/en/blog/new-in-chrome-87/index.md:19]: Done.

**Comment by petele** [comment]
@robdodson - this one is ready to go!

**Review by robdodson — APPROVED** [review]

**Pull request merged**